src/Console/Command/FixCommand.php

```diff
@@ -292,8 +294,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : null;
-        if ($stdErr && extension_loaded('xdebug')) {
+        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : $output;
+        if (extension_loaded('xdebug')) {
```

Comment:
@SpacePossum , this must not be printed to stdout on not txt format, as it would break JSON/YML format.

Other changes in this commit are OK.